src/style.css

```diff
@@ -0,0 +1,131 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+}
+
+.form {
+  position: relative;
+  box-sizing: border-box;
+  width: 100%;
+  border: none;
+}
+
+.formBig {
+  margin: 20px 0;
+  line-height: 22px;
+  height: 70px;
+}
+
+.formSmall {
+  height: 42px;
+  line-height: 19px;
+}
+
+.search {
+  position: relative;
+  box-sizing: border-box;
```

Comment:
It is a good tone to set this property for all elements(*) and don't change without extra need

Reply:
Thanks for your advice!
I'll keep that in mind next time)